canonical-web-and-design/design.ubuntu.com
# Remove pictograms, update Vailla

## Done

- Updated to Vanilla 2.10.1 which fixes the font issues
    - Updated the header
    - Updated the footer
    - Updated the link to the web & design blog
- Remove the pictograms
    - Left the pictogram page, but remove the link to the downloads of pictograms

## QA

1. Download this branch
2. ./run clean && ./run the site
3. Look at http://0.0.0.0:8011/downloads/ and see that the pictograms are removed
4. Look at http://0.0.0.0:8011/font and see the fonts look ok

## Issue / Card

Fixes #150 and #151
